data/json/furniture_and_terrain/terrain-floors-indoor.json

```diff
@@ -1719,11 +1720,11 @@
     "name": "open air",
     "description": "An area of open air.",
     "//": "Actually, this is a thin membrane of bullshit to keep wasp eggs levitating.",
+    "//1": "Why do we want eggs to levitate??",
```

Comment:
I think this is because they're actually supported from *above*.

Reply:
Which is something the current support logic doesn't support (nor are overhangs, which doesn't stop JSON specified buildings to have them), so some form of hand waving is needed.